packages/contracts-bridge/contracts/test/BridgeRouter.t.sol

```diff
@@ -124,7 +130,7 @@ contract BridgeRouterTest is BridgeRouterBaseTest {
         emit Transfer(address(0), recipient, tokenAmount);
```

Comment:
we should probably replace the `expect emit Transfer` on the ERC20 token with `expectCall` to mint, but - non-blocking comment

Reply:
in general, I replaced only things that were required to make the test function in fork mode